hedera-node/src/test/java/com/hedera/services/context/AwareTransactionContextTest.java

```diff
@@ -246,21 +250,21 @@ public void getsPayerKeyIfSigActive() {
 	}
 
 	@Test
-	public void getsExpectedNodeAccount() {
+	void getsExpectedNodeAccount() {
 		// expect:
 		assertEquals(nodeAccount, subject.submittingNodeAccount());
 	}
 
 	@Test
-	public void failsHardForMissingMemberAccount() {
-		given(book.getAddress(memberId)).willReturn(null);
+	void failsHardForMissingMemberAccount() {
+		given(nodeInfo.accountOf(memberId)).willThrow(IllegalArgumentException.class);
 
 		// expect:
 		assertThrows(IllegalStateException.class, () -> subject.submittingNodeAccount());
```

Comment:
Same here, should add an expectation for the `IllegalStateException.getMessage()`
Also expect a `log.warn()` and assert its message as well.

Reply:
Added assert on `log.warn`/message.